src/app/multiselect/multiselect.factory.js

```diff
@@ -13,7 +13,14 @@
      */
     function MultiselectFactory($parse) {
 
-        var _optionsRegularExpression = /^\s*(?:(\S+)\s+as\s+)?(\S+)\s+for\s+(\S+)\s+in\s+(\S+)\s*$/;
+        var _optionsRegularExpression = /^\s*(?:(\S+)\s+as\s+)?(\S+)(?:\s+group\s+by\s+([\s\S]+?))?\s+for\s+(\S+)\s+in\s+(\S+)\s*$/;
```

Comment:
Is there a reason why the character class (`[\s\S]`) for the group includes whitespace?